architecture/azure/messaging.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Azure messaging
+summary:
+reviewed: 2023-07-18
+callsToAction: ['solution-architect', 'poc-help']
+---
+
+Azure offers [several messaging services](https://learn.microsoft.com/en-us/azure/service-bus-messaging/compare-messaging-services), each built for specific purposes. The Particular Service Platform supports [Azure Service Bus and Azure Queue Storage](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-azure-and-service-bus-queues-compared-contrasted).
```

Comment:
Article linked calls them "storage queues" - any reason why this is in the opposite order?

Reply:
yeah microsoft doesn't seem to be consistent with it either, e.g. https://learn.microsoft.com/en-us/azure/storage/queues/storage-queues-introduction calls it Azure Queue Storage 🤷